cpp/include/cuml/fil/fil.h

```diff
@@ -91,6 +91,7 @@ struct sparse_node_t {
   float val;
   int bits;
   int left_idx;
+  int dummy;
```

Comment:
I would add a comment next to dummy like `// Pad to X bytes for better cache packing` or whatever is more descriptive so others know why this exists.

Reply:
Done.